core/node/da_clients/src/eigen/client.rs

```diff
@@ -9,37 +9,27 @@ use zksync_da_client::{
     DataAvailabilityClient,
 };
 
-use super::sdk::RawEigenClient;
+use super::{memstore::MemStore, sdk::RawEigenClient, Disperser};
 use crate::utils::to_non_retriable_da_error;
 
 #[derive(Debug, Clone)]
 pub struct EigenClient {
-    client: Arc<RawEigenClient>,
+    client: Disperser,
```

Comment:
It may be a good idea to rename this field to `disperser`:
```suggestion
    disperser: Disperser,
```

Reply:
In order to make the changes smaller will leave it as is